cli/src/errors.rs

```diff
@@ -0,0 +1,15 @@
+#[derive(Debug, thiserror::Error)]
+pub enum Errors {
+    #[error("Config error: {0}")]
+    ConfigError(#[source] config::ConfigError),
```

Comment:
I didn't use #[source] but you can use #[from] which automatically implement From trait and you can use more simply code, for example:
```
        .send()
        .await?
        .json::<DeleteScript>()
        .await?;
```
instead 
```
.await
        .map_err(Errors::ReqwestError)?
        .json::<DeleteScript>()
        .await
        .map_err(Errors::ReqwestError)?;
```